### ISSUE TITLE
TypeError: Cannot read property 'bind' of undefined

### ISSUE THREAD

**Issue by eldadts** [xiaokaike/vue-clipboard#8]
I tried this with vue 2.5.2 and I get this error:

`TypeError: Cannot read property 'bind' of undefined
    at bind (vue-clipboard.js?59e1:12)`

**Comment by BHerbulot** [comment]
Same problem here

**Comment by matthew-inamdar** [comment]
Same problem here

**Comment by sebastiansulinski** [comment]
Yep - I get `Error in directive clipboard bind hook: "TypeError: Cannot read property 'bind' of undefined"`

**Comment by bahmanenayati** [comment]
Same problem here :?

**Comment by DJanoskova** [comment]
Me too, trying to bind it with ES6 syntax in main.js and PHPStorm says Namespace 'v-clipboard' is not found because we are not registering it as a directive, as "v-" stuff should be

`import VueClipboard from 'vue-clipboard'
Vue.use(VueClipboard)`

**Comment by norbertdragan** [comment]
Same issue.
